python/xorbits/web/ui/src/node_info/NodeResourceTab.js

```diff
@@ -61,25 +61,74 @@ export default class NodeResourceTab extends React.Component {
 
   generateBandRows() {
     const converted = {}
+
+    const tablecontentReturn = (converted, band) => {
```

Comment:
Should this be ``tableContentReturn`` (Hump Naming) ? And just ``tableContent`` is fine.

Reply:
refactored in the latest version